addons/html_editor/static/tests/list/list_font_size.test.js

```diff
@@ -193,3 +193,12 @@ test("should pad list based on font-size (2)", async () => {
         contentAfter: `<ol style="padding-inline-start: 60px;"><li style="font-size: 56px;">[]a</li></ol>`,
     });
 });
+
+test.tags("font-dependent");
+test("should apply color to a list containing sublist if list contents are fully selected", async () => {
+    await testEditor({
+        contentBefore: "<ol><li><p>[abc]</p><ol><li>def</li></ol></li></ol>",
+        stepFunction: setFontSize("56px"),
+        contentAfter: `<ol style="padding-inline-start: 60px;"><li style="font-size: 56px;"><p>[abc]</p><ol><li>def</li></ol></li></ol>`,
+    });
+});
```

Comment:
I don't understand. Why does this behave differently than color? I would expect the font size to be applied only on the selected LI, just like the previous changed test in this commit.

Also, in any case, this will need to be adapted with https://github.com/odoo/odoo/pull/212838 in mind.

Reply:
Indeed it's the same. I don't understand how I read it any other way, sorry about that.